source/iterator/stream_iterator.go

```diff
@@ -88,7 +93,7 @@ func NewStreamIterator(ctx context.Context,
 }
 
 // HasNext returns whether there are any more records to be returned
-func (i *StreamIterator) HasNext(_ context.Context) bool {
+func (i *StreamIterator) HasNext() bool {
```

Comment:
So, this comment is not related to this line, but I couldn't find a better place in this PR for this question: When you use `XREAD` with `COUNT`, will the command wait in the `COUNT` parameter is more than the number of available messages? For example: let's say a stream has 20 messages, but you have `XREAD COUNT 100`. Are you going to get those 20, or will it block? I couldn't it in a quick search in the docs, but I guess it's the former.

Reply:
@aathiraashok ^ in case you maybe missed the above.